flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/model/Property.java

```diff
@@ -199,10 +201,17 @@ public <T> T valueAsType(TypeToken<T> typeToken) {
     public static final String RETRY_COUNT_LABEL = "Retry Count";
     public static final String RETRY_COUNT_DOC = "Number of retries";
 
-    public static final String DEFAULTABLE_KEY = "defaultable";
-    public static final String DEFAULT_VALUE_LABEL = "Default value";
-    public static final String DEFAULT_VALUE_DOC = "Default value for the config, if empty your need to " +
-            "provide a value at runtime";
+    public static final String DEFAULT_VALUE_KEY = "defaultValue";
+    public static final String DEFAULT_VALUE_LABEL = "Default Value";
+    public static final String DEFAULT_VALUE_DOC = "Default value of the variable";
+
+    public static final String CONFIG_VALUE_KEY = "configValue";
+    public static final String CONFIG_VALUE_LABEL = "Config Value";
+    public static final String CONFIG_VALUE_DOC = "Config value of the variable, to be used in Config.toml";
+
+    public static final String CONFIG_VAR_DOC_KEY = "documentation";
+    public static final String CONFIG_VAR_DOC_LABEL = "Description";
+    public static final String CONFIG_VAR_DOC_DOC = "Documentation for the variable, provided in Markdown format.";
```

Comment:
```suggestion
    public static final String CONFIG_VAR_DOC_DOC = "Documentation of the variable, provided in Markdown format";
```

Reply:
Simplified as `Variable documentation in Markdown format`. Added in https://github.com/ballerina-platform/ballerina-language-server/commit/3cac34ad38d9baf68002172e467c44757acf2042